ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -22,6 +22,7 @@ ${DSCI_NAME} =    default-dsci
 ${SERVERLESS_OP_NAME}=     serverless-operator
 ${SERVERLESS_SUB_NAME}=    serverless-operator
 ${SERVERLESS_NS}=    openshift-serverless
+${OPERATORS_NS}=    openshift-operators
```

Comment:
could you change the name of the variable something like `OPENSHIFT_OPERATORS_NS` . This is because to avoid confusion between the rhoai operator ns and other as of today we use `OPERATOR_NAMESPACE` for `redhat-ods-operator`

Reply:
done @tarukumar !